stable/traefik/README.md

```diff
@@ -112,8 +112,9 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `acme.persistence.accessMode`   | `ReadWriteOnce` or `ReadOnly`                                        | `ReadWriteOnce`                           |
 | `acme.persistence.size`         | Minimum size of the volume requested                                 | `1Gi`                                     |
 | `dashboard.enabled`             | Whether to enable the Traefik dashboard                              | `false`                                   |
-| `dashboard.domain`              | Domain for the Traefik dashboard                                     | `traefik.example.com`                     |
-| `dashboard.ingress.annotations` | Annotations for the Traefik dashboard Ingress definition, specified as a map | None                              |
+| `dashboard.domain`              | Domain for the Traefik dashboard                                     | `traefik.example.com`                     |               
+| `dashboard.service.annotations` | Annotations for the Traefik dashboard Service definition, specified as a map | None                |
```

Comment:
In the values.yaml file `dashboard` and `service` are at the same level rather than `service` being under dashboard. This descriptions doesn't appear to match.

Reply:
Hi @mattfarina, the service that this PR is referring to is for Traefik's dashboard, not for Traefik itsself, thus the parameter matches!

Reply:
@mattfarina to avoid confusion - this is an example usage of the `dashboard.service.annotations` parameter:

<img width="710" alt="screen shot 2018-01-09 at 16 57 48" src="https://user-images.githubusercontent.com/101739/34732844-56a583a2-f55e-11e7-8e86-14b6db8d64b2.png">

Reply:
In the `values.yaml` file there isn't a section for `dashboard.service`. See:

https://github.com/kubernetes/charts/blob/e938d4f33e1e8150de430ee523d26b8c1b18b549/stable/traefik/values.yaml#L51-L80

In the `values.yaml` file both `dashboard` and `service` are at the same level.

I see that you're adding annotations to the dashboard service. That section of the `values.yaml` file should be updated to provide for the annotations.

Reply:
@mattfarina aha, thanks for clarifying 😄 I've added this change now.